### ISSUE TITLE
Empty alt attribute is not consistently supported - No longer true

### ISSUE THREAD

**Issue by giacomo-petri** [auto-wcag/auto-wcag#2273]
## Accessibility support

> There are several popular browsers that do not treat images with an empty alt attribute (alt="") as having a role of presentation but instead add the img element to the accessibility tree with a [semantic role](https://www.w3.org/WAI/standards-guidelines/act/rules/23a2a8/#semantic-role) of either img or graphic.

## ACT Rules

- [Image has non-empty accessible name - 23a2a8](https://www.w3.org/WAI/standards-guidelines/act/rules/23a2a8/#accessibility-support)

### Test case 1

An image with empty alt attribute value or no alt attribute value at all

```html
<img src="some_src.png" alt="">
<img src="some_src.png" alt>
```

[Testcase](https://codepen.io/giacomo-usablenet/full/ogvrmRq)

### External links
- WPT: https://wpt.fyi/results/html-aam/roles.html?label=experimental&label=master&aligned ['el-img-alt-no-value', 'el-img-empty-alt']
- ARIA in HTML: https://www.w3.org/TR/html-aria/#el-img-no-name

## Test results

All browsers are ignoring/treating the image with empty alt as generic
